docs/connection-and-channel-management.md

```diff
@@ -0,0 +1,53 @@
+# Connection and channel management
+
+Spaces SDK uses the Ably SDK client [connections](https://ably.com/docs/connect?lang=javascript) and [channels](https://ably.com/docs/channels?lang=javascript) to provide higher level features like cursors or locations. Both connections and channels will transition through multiple states throughout their lifecycle. Most state transitions (like a short loss in connectivity) will be handled by the Ably SDK, but there will be use cases where developers will need to observe these states and handle them accordingly.
+
+There is no dedicated Spaces API to handle the connection or channels, however the relevant Ably SDK APIs are available. This document describes how to access a connection and channels on Spaces, and where to find information about how to handle their state changes.
```

Comment:
I don't think this is the right message.

We're not saying there is no API, there is one.  It just happens to be a proxy to the underlying connection or channel. That is a design decision we are making, and that's OK.  

The message makes it sound like this is a shortcoming